KarhooUISDK/Screens/QuoteList/QuoteListErrorView.swift

```diff
@@ -0,0 +1,130 @@
+        $0.textColor = KarhooUI.colors.text
+        $0.textAlignment = .center
+        $0.numberOfLines = 0
+        if let attributedMessage = viewModel.attributedMessage {
+            $0.attributedText = viewModel.attributedMessage
+        } else {
+            $0.text = viewModel.message
+        }
+    }
+
+    private lazy var imageView = UIImageView().then {
+        $0.image = .uisdkImage(viewModel.imageName)
+        $0.contentMode = .scaleAspectFit
+    }
+
+    private lazy var contentStackView = UIStackView().then {
+        $0.axis = .vertical
+        $0.alignment = .center
+    }
+
+    // MARK: - Properties
+
+    let viewModel: QuoteListTableErrorViewModel
+    private weak var delegate: QuoteListErrorViewDelegate?
+
+    // MARK: - Initialization
+
+    init(using viewModel: QuoteListTableErrorViewModel, delegate: QuoteListErrorViewDelegate?) {
+        self.viewModel = viewModel
+        self.delegate = delegate
+        super.init(frame: .zero)
+        self.setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+
+    override func didMoveToWindow() {
+        super.didMoveToWindow()
+        animateIn()
+    }
+
+    // MARK: - Setup
+
+    private func setupView() {
+        addSubview(contentStackView)
+        contentStackView.anchorToSuperview(padding: UIConstants.Spacing.standard)
+        contentStackView.addArrangedSubviews([
+            buildSeparator(),
+            imageView,
+            buildSeparator(height: UIConstants.Spacing.standard),
+            titleLabel,
+            buildSeparator(height: UIConstants.Spacing.standard),
+            descriptionLabel,
+            buildSeparator()
+        ])
+        imageView.anchor(width: Constants.imageViewSize.width, height: Constants.imageViewSize.height)
+        imageView.centerXAnchor.constraint(equalTo: centerXAnchor).isActive = true
+        imageView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
+    }
+
+    // MARK: - Helpers
+
+    private func animateIn() {
+        [
+            imageView,
+            titleLabel,
+            descriptionLabel
+        ].forEach(aniamteSubviewIn)
+    }
+
+    private func aniamteSubviewIn(_ viewToAnimate: UIView) {
+        viewToAnimate.alpha = UIConstants.Alpha.hidden
+        viewToAnimate.transform = CGAffineTransform(scaleX: 0.5, y: 0.5)
+        UIView.animate(
+            withDuration: UIConstants.Duration.medium,
+            delay: .random(in: (0...UIConstants.Duration.xSmallDelay)),
+            usingSpringWithDamping: UIConstants.Animation.springWithDamping,
+            initialSpringVelocity: UIConstants.Animation.initialSpringVelocity,
+            options: .curveEaseIn,
+            animations: {
+                viewToAnimate.alpha = UIConstants.Alpha.enabled
+                viewToAnimate.transform = .identity
+            },
+            completion: nil
+        )
+    }
+
+    private func buildSeparator(height: CGFloat? = nil) -> UIView {
```

Comment:
should be in separate file to make it reusable in another stacks. Compare to LineView and maybe reorganize them to VarticalLine and HirizontalLine views or something like that.